docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,206 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Quick Tour"
+  weight = 2
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour
+
+Morphia wraps the mongodb Java driver so some level of familiarity with using the driver can be helpful.  Morphia does its best to 
+abstract much of that away but if something is confusing it wouldn't hurt to consult the Java driver [documentation](http://mongodb.github
```

Comment:
"but if something is confusing it wouldn't hurt" is a bit colloquial for reference documentation.   And if it's confusing, let's make it unconfusing by either explaining it or linking to the appropriate specific documentation from the java driver.